### PR TITLE
Fix #3757. Image copied in clipboard is not insterted as base64 when not using uploadimage plugin

### LINKED ISSUE #3757: Image copied in clipboard is not insterted as base64 when not using uploadimage plugin
## Are you reporting a feature request or a bug?

Feature request

## Provide detailed reproduction steps (if any)

1. We are not using uploadimage, nor easyimage
2. Press print screen
3. Paste with Ctrl+V in ckeditor

### Expected result

Image must be inserted as base64 like as when I copy/paste a file image

### Actual result

Nothing when uploadimage not used

## Other details

* Browser: Firefox 71
* OS: Windows
* CKEditor version: 4.13.1
* Installed CKEditor plugins: base64image and no  uploadimage, nor easyimage

### PR DESCRIPTION
**Pull request by albfernandez**
## What is the purpose of this pull request?
Fix a regression pasting images in Firefox, view #3757

In 4.13.0 and before you can paste an image in standard Ckeditor with Firefox and that image was inserted as base64. In 4.13.1 this functionality was lost (by changes in  pull request #3639)

## Does your PR contain necessary tests?

All patches that change the editor code must include tests. You can always read more
on [PR testing](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_contributing_code.html#tests),
[how to set the testing environment](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html) and
[how to create tests](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

Sorry, no test included.
I've manually test:
  * pasting images in Firefox was recovered, 
  * fix for pasting Office contents (#3639) is working.

## Did you follow the CKEditor 4 code style guide?

Your code should follow the guidelines from the [CKEditor 4 code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md) which helps keep the entire codebase consistent.

- [x] PR is consistent with the code style guide

## What is the proposed changelog entry for this pull request?

```
* [#3757](https://github.com/ckeditor/ckeditor4/issues/3757): Fix pasing images as base64 in Firefox
```

## What changes did you make?
Pass a lost parameter "latestId", and allow pasting as image if and empty types was provided (I checked it by debugging the code when pasting an image) 

*Give an overview…*

## Which issues does your PR resolve?

Closes #3757.

### CODE REVIEW

**Review by Comandeer — CHANGES_REQUESTED** [review]
Thank you for your contribution!

The fix works, but it seems that the issue is deeper and should be fixed by enhancing the way CKEditor gets types from pasted content (see #4604). Because of that, I will mark this PR as blocked until the mentioned issue is resolved.

Despite that this PR lacks unit and manual tests. They will be needed anyway to check if the issue is fixed. Can I ask you to add some? You can check out our [existing tests for the `clipboard` plugin](https://github.com/ckeditor/ckeditor4/tree/major/tests/plugins/clipboard) as an example. There are also [docs describing how to create your own test](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test). The test case should invoke `paste` event with appropriate data (so `DataTransfer` containing only one file) and check if the content inside the editor is updated correctly to include Base64 encoded image.

So to sum up:
* the fix itself should be revisited after #4604 is resolved,
* tests can be added to this PR while #4604 is being fixed.

**Comment by Comandeer** [comment]
Once again thanks for your contribution! However we decided to use a slightly different solution (#4662).
